### DISCUSSION TITLE
Scalar is "unkown"

### DISCUSSION THREAD

**Discussion by ThomasKientz** [0no-co/gql.tada#395] [Q&A] [answered]
Hi,

In the following query, in my understanding, the "IBAN" fields returns a scalar of type IBAN. But it appears as `unkown` in typescript. 

Field `id` is scalar as well but properly typed as `string`

```typescript
const query = graphql(`
    query GetAccount($id: ID!) {
      account(accountId: $id) {
        IBAN
        id
      }
    }
  `);
```
<img width="570" alt="Capture d’écran 2024-09-18 à 16 07 40" src="https://github.com/user-attachments/assets/5ced75ef-7f0b-4e40-8c9a-c73cb15825f0">

Here is a screenshot of the playground : 

<img width="1152" alt="Capture d’écran 2024-09-18 à 16 03 46" src="https://github.com/user-attachments/assets/d1180e64-277c-40f0-af50-e399e824b0a7">

Am I missing something  ? I am using swan's api (https://explorer.swan.io/).

[graphql-env.d.ts.zip](https://github.com/user-attachments/files/17045191/graphql-env.d.ts.zip)

**Comment by kitten** [comment, marked as answer]
Have you defined a type for the `IBAN` scalar? https://gql-tada.0no.co/get-started/installation#customizing-scalar-types

**Reply by ThomasKientz** [reply]
Thanks... As I havent for the `ID` scalar, I thought all scalars were auto-generated by default.
